### ISSUE TITLE
Build fails if nested directory tree is missing

### ISSUE THREAD

**Issue by wallzero** [drolsen/webpack-favicons#8]
I am building multiple pages in a monorepo and the output would be something like `public/**PROJECT NAME**`. However, `public` and the sub folders may not exist at initial build time. Webpack creates the directories as it completes; but `webpack-favicons` throws an error if the build target directory is missing:

```
[webpack-cli] Error: ENOENT: no such file or directory, mkdir 'public/**PROJECT NAME**/bundle'
```

**Comment by drolsen** [comment]
Apologies for the delay, this seemed to had fallen off my radar.
I've now included the `{ recursive: true }` option on fs.mkdirSync to resolve this.

Thanks!

**Comment by wallzero** [comment]
Thanks! I'll just mention I had to include a trailing '/' in my `path` or else Webpack just prefixed the `path` to the favicon filename.

**Comment by drolsen** [comment]
Good point.
I've just released a 1.2.5 version that now pads `options.path` and  `file.name` with extra `/` and then is ran through path.normalize. This ensure no matter the configuration's leading or trailing slash pattern(s), it will not be joined with filename. 

I also updated the pre-pended `output.publicPath` with this same treatment to again assure no pattern preference renders different results.
